### ISSUE TITLE
Style/Filename: use snake_case for Gemfile

### ISSUE THREAD

**Issue by amingilani** [AtomLinter/linter-rubocop#163]
Almost exactly as #2, I get `Style/Filename` errors to use `snake_case` on the Gemfile of my Rails project, a false positive.

<img width="725" alt="screen shot 2016-09-19 at 7 35 42 am" src="https://cloud.githubusercontent.com/assets/1188552/18621364/b6beefd0-7e3b-11e6-90dd-94ea93eada7c.png">
## Additional tests

```
$ /Users/gilani/.rvm/gems/ruby-2.3.1/wrappers/rubocop
Inspecting 96 files
................................................................................................

96 files inspected, no offenses detected
```
## System Details

My `rubocop` executable path in the package settings is set to `/Users/gilani/.rvm/gems/ruby-2.3.1/wrappers/rubocop` which produces no errors, as shown above.

The same path is also set in `atom-beautify`'s rubocop path.

```
rvm 1.27.0
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin15]
$ apm -v
apm  1.12.5
npm  3.10.5
node 4.4.5
python 2.7.12
git 2.10.0
$ atom -v
Atom    : 1.10.2
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0
rubocop 0.42.0

linter-rubocop 0.4.7
OSX 10.11.6
```

**Comment by Arcanemagus** [comment]
Since it looks like you are using RVM, does it work if you close _all_ instances of Atom, then launch it from a terminal in that project where `rubocop` works from the CLI (`atom .`)?

**Comment by amingilani** [comment]
@Arcanemagus 

Nope.

I have a habit of using the command line to navigate to my project directory and then using `atom .` to open it.

I also just tried opening atom outside the project directory and using `File> Open` to open the file, and the problem still persists.

**Comment by amingilani** [comment]
Sorry, I accidentally clicked the wrong button. Opened the issue again.

**Comment by effkay** [comment]
Just ran into this setting up a fresh dev env. But it seems to be a rubocop and not a rubocop-linter issue? See https://github.com/bbatsov/rubocop/issues/2973

**Comment by amingilani** [comment]
@effkay not in this case. I ran `rubucop` using the exact same command as specified in `linter-rubocop`, and didn't run into any errors.

**Comment by amingilani** [comment]
We can also rule out `rvm` related issues. I just imploded rvm and reinstalled the entire thing from scratch.

**Comment by ChallahuAkbar** [comment]
Bumping, because I've just recently started having the same issue.

**Comment by maschwenk** [comment]
I made the Gemfile parsing more specific, so this should be fixed by: https://github.com/AtomLinter/linter-rubocop/compare/v2.1.1...v2.2.0#diff-1fdf421c05c1140f6d71444ea2b27638R172
